source/README.md

```diff
@@ -4,12 +4,26 @@ Redis Event Source for Knative
 
 ## Getting started
 
+Prior to installing the source, you will need to decide whether you need to add your TLS Certificate that lets you securely connect to your Redis instance (for example, an instance of Redis on IBM Cloud) to the [`config-tls`](config/config-tls.yaml) Config Map. You do not need to do this if you are using a local instance of Redis, as described in the Example section below.
```

Comment:
@aavarghese The user isn't really deciding whether they need to add TLS Certificate or not -- they're deciding whether they want to use a local or remote/cloud instance of redis. Can we rephrase this?  Maybe something like:
"If you are using a local Redis installation, you can skip this step. If you would like to use a cloud installation of Redis, you will need to add the TLS certificate etc etc"

Reply:
Done (commit bc8d7bacabd96f4817c9eef87ade03ecac171df1)